.github/workflows/pathogen-repo-ci.yaml

```
@@ -196,7 +196,8 @@ jobs:
           '
           | tee -a "$GITHUB_ENV"
 
-      - uses: actions/checkout@v4
+      - name: Checkout repo
```

> Add names to all steps

`actions/checkout` is so common that we've typically not named it, unless there are multiple checkouts/clones to disambiguate. Since that's the case here (in a subsequent commit), consider something like?

```suggestion
      - name: Checkout pathogen repo
```

---

Changed to `Checkout ${{ inputs.repo }}`